javascript/attributes.js

```diff
@@ -62,6 +64,38 @@ export const extractElementAttributes = element => {
       }
     }
   }
+  return attrs
+}
+
+// Extracts dataset of an element and includes inherited data attributes from parents if requested.
+//
+export const extractElementDataset = element => {
+  let attrs = extractDataAttributes(element) || {}
+
+  if (element.attributes[defaultSchema.reflexInheritAttribute]) {
```

Comment:
We can't use `defaultSchema` here because consumers of the library may change the name of this attribute.

We'll need to update to use `stimulusApplication.schema`. There are a few ways to accomplish this, but I'll let you experiment and find something that works well.

Reply:
TIL that you can rename the `data-reflex-*` attributes 😉